### PR TITLE
Move the i18n rest endpoint under "rest"

### PR DESCRIPTION
**Pull request by tfennelly**
Keeping my buddy @vivek happy !!

@reviewbybees

### CODE REVIEW

File: blueocean-core-js/src/js/i18n/i18n.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ function newPluginXHR(pluginName) {
 
     pluginVersion = encodeURIComponent(pluginVersion);
 
-    const loadPath = `${prefix}/blueocean-i18n/${pluginName}/${pluginVersion}/{ns}/{lng}`;
+    const loadPath = `${prefix}/blue/rest/i18n/${pluginName}/${pluginVersion}/{ns}/{lng}`;
```
Comment (vivek) [inline blueocean-core-js/src/js/i18n/i18n.js]: Perhaps could use JS variable to get base of API rest url.

**Comment by vivek** [comment]
LGTM 🐝

**Comment by vivek** [comment]
LGTM 🐝

**Comment by michaelneale** [comment]
looks good - need to check with ATH and also need to not use a beta

**Comment by michaelneale** [comment]
This passed 2x in a row on ATH https://ci.blueocean.io/job/ATH-Jenkinsfile/job/rollback-test/12/ 

if beta can be resolved then :bee: for this and that ATH branch

**Pull request merged**